README.md

```diff
@@ -1,53 +1,76 @@
 # Climate Connect
 
-The code for https://climateconnect.earth. 
+The code for https://climateconnect.earth.
 
 ## Pre-steps
 
-1.  Create a local Postgres database with your own username and password. You will connect to this for your local backend project.
-2.  Create 32 char random secret key for your local setup. Run `openssl rand -base64 32`. Copy the
-    value you will be using this to add as SECRET_KEY for django settings.
+### Postgres
 
-## Getting Started Locally
+Create a local Postgres database with your own username and password. You will connect to this for your local backend project.
 
-We are using Python/Django for backend and Next.js for frontend.
+### Docker
 
-1.  Create a python virtual environment `python3 -m venv climateconnect_env`
-2.  Run following commands to start virtual environment
-    ```
-       cd climateconnect_env
-       source bin/activate
-    ```
-3.  Clone GitHub repository `git clone <URL>`
+We use Docker to run the local Redis server. See the [Docker install docs](https://docs.docker.com/get-docker/) if you don't have it.
 
-### Run docker compose
+## Get Started
 
-Currently, we use docker to run Redis server locally. 
+We use Python/Django for our backend and Next.js for the frontend.
 
-If you do not have docker installed locally, follow the official steps: https://docs.docker.com/get-docker/
+First, create a Python virtual environment and start it
 
-1. Go to backend directory `cd backend`
-2. Run `docker-compose up`. This will start Redis server on docker
+```sh
+python3 -m venv climateconnect_env
+cd climateconnect_env
+source bin/activate
+```
+
+Then clone the GitHub repository
+
+```sh
+git clone https://github.com/climateconnect/climateconnect
+```
 
 ### Backend
 
-Once you clone the repository start backend server.
+After you've cloned the repository, we can set up the local Redis server and backend.
+
+Note: we use Python 3, so for all instructions we insume `python` means `python3`.
+
+#### First Time Setup
+1.  Go to backend directory: `cd backend`
+1.  Run `pip install -r requirements.txt` to install all backend libararies.
```

Comment:
Just found one more small error that we can directly fix in this PR: "libraries" --> "libraries"